src/uint/sqrt.rs

```diff
@@ -65,6 +79,8 @@ impl<const LIMBS: usize> Uint<LIMBS> {
 
 #[cfg(test)]
 mod tests {
+    #![allow(deprecated)]
```

Comment:
Why are you testing the deprecated names?

Reply:
Sorry, I wasn't sure about the procedure. So we're adding a new name without removing the old one, correct?

Reply:
Looks like you got everything updated.